feconf.py

```diff
@@ -33,6 +33,10 @@
 IS_MINIFIED = os.environ.get('MINIFICATION') == 'True'
 
 # Whether we should serve the development or production experience.
+# This is environment specific variable(DEV_MODE) changes to true only
```

Comment:
Here's an edited version: "DEV_MODE should only be changed to False in the production environment. To use minified resources in the development environment, change the MINIFICATION env variable in app.yaml to True."

@MAKOSCAFEE -- I still don't understand what DEV_MODE is, though (I don't know what you mean by an "environment-specific variable"). Can you perhaps explain what it affects?